ui/components/app/signature-request-original/signature-request-original.container.js

```diff
@@ -80,54 +85,48 @@ function mapDispatchToProps(dispatch) {
         }),
       );
     },
-    cancelAll: (messagesList) => dispatch(cancelMsgs(messagesList)),
+    resolvePendingApproval: (id) => {
+      dispatch(resolvePendingApproval(id));
+    },
+    rejectPendingApproval: (id, error) =>
+      dispatch(rejectPendingApproval(id, error)),
+    cancelAllApprovals: (messagesList) => {
```

Comment:
Rather than defining this inline and doing multiple dispatches, could we move this to the `actions` so we can dispatch once, something like `rejectAllMessages`, potentially using a generic `rejectPendingApprovals`?

Reply:
Fixed, added `rejectAllMessages` and consumed on both signature containers.